inheritance/src/main/java/inheritance/Theater.java

```diff
@@ -0,0 +1,31 @@
+package inheritance;
+
+import java.util.ArrayList;
+
+public class Theater {
+    public String name;
+    public ArrayList<Review> reviews = new ArrayList<>();
+    public ArrayList<String> movies = new ArrayList<>();
+
+    public Theater(String name) {
+        this.name = name;
+    }
+
+    public void addReview(Review review) { reviews.add(review); }
+
+    public void addMovie(String movie) {
+        movies.add(movie);
```

Comment:
Food for thought: could you have coded a way to prevent users from adding the same movie more than once?